modules/gui/emulator_controls.py

```diff
@@ -167,7 +166,9 @@ def _update_stats(self):
         current_load = context.emulator.get_current_time_spent_in_bot_fraction()
         if current_fps:
             stats.append(f"{current_fps:,}fps ({current_fps / 60:0.2f}x)")
-        stats.append(f"{get_encounter_rate():,}/h")
+        if context.profile:
+            from modules.stats import total_stats  # TODO prevent instantiating TotalStats class before profile selected
```

Comment:
This was being imported before a profile was selected, due to `from modules.gui.debug_tabs import *` in `emulator_screen.py`, these imports should probably be deferred until a profile is selected